packages/server-core/src/media/static-resource/static-resource.ts

```diff
@@ -57,4 +60,29 @@ export default (app: Application): void => {
 
   const service = app.service(staticResourcePath)
   service.hooks(hooks)
+
+  const onCRUD = (data: StaticResourceType | Paginated<StaticResourceType> | StaticResourceType[]) => {
+    const targetIds: string[] = []
+    if (Array.isArray(data)) {
+      data.forEach((item) => {
+        targetIds.push(item.userId)
```

Comment:
I dont think userId does much here.

This will trigger refresh in client only for user who uploaded that static-resource.

Maybe we can update onCRUD method to match our needs at the moment. Which is it should only triggered when static-resource type's type column was `scene`.

Further more the targetIds which be evaluated based on the project that scene was created and then, project permissions should be used to find userId like its done in project.ts

Reply:
Fixed!